hadoop-client-modules/hadoop-client-integration-tests/pom.xml

```diff
@@ -57,6 +57,11 @@
       <artifactId>lz4-java</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
+      <scope>test</scope>
```

Comment:
If the hadoop client jar already depends on snappy-java transitively, why do we need to add it to test scope?

Reply:
Because we exclude it from the uber jar, we don't include the artifact of snappy-java in client libraries like hadoop-client-runtime. Note that both hadoop-client-runtime and hadoop-client-api make hadoop-client as optional. So it won't count hadoop-client as transitive dependency of these client libraries.